integration_tests/mockApis/sendLegalMail.ts

```diff
@@ -306,6 +306,54 @@ const stubCreateBarcodeFailure = (): SuperAgentRequest =>
     },
   })
 
+const stubCreateBarcodeSuccessThenCreateBarcodeFailure = (): Promise<unknown> => {
+  return Promise.all([
+    stubFor({
+      scenarioName: 'createBarcodeThenFail',
```

Comment:
This is neat - you can setup a wiremock scenario (which is really just a collection of related requests), where each request has a required start state. The initial wiremock state is `Started`, and then you can trigger a new state when the request executes.
This means you can setup wiremock to respond to 2 successive calls to the same endpoint with different responses - perfect for saying "the call to generate a barcode value for the first recipient will work, but the call for the second recipient will fail" 👍